cli/src/pcluster/config/cfn_param_types.py

```diff
@@ -268,8 +268,6 @@ def get_value_from_string(self, string_value):
         """Return internal representation starting from CFN/user-input value."""
         param_value = self.get_default_value()
         try:
-            # Do not convert empty string and use format and yaml.load in place of json.loads
-            # for Python 2.7 compatibility because it returns unicode chars
             if string_value:
```

Comment:
Minor: should we also refactor this method since  Python 2.7 has been removed, but it's fine if we keep this logic, because it doesn't affect functionality.

Reply:
Yep I am for keeping the current implementation